src/components/HomePage.js

```diff
@@ -1,24 +1,27 @@
+/* @flow */
+import type { SessionState } from "../types";
+
 import React, { Component } from "react";
 
 import Spinner from "./Spinner";
 import AuthForm from "./AuthForm";
 
 
-function ServerProps({serverInfo}) {
+function ServerProps({node}: {node: Object}) {
```

Comment:
Note: this is left vaguely typed on purpose as this is a recursive component which deals with differently shaped nodes.